SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -53,6 +53,34 @@ DEFINE_STAT(STAT_SpatialConsiderList);
 DEFINE_STAT(STAT_SpatialActorsRelevant);
 DEFINE_STAT(STAT_SpatialActorsChanged);
 
+namespace
+{
+	void ConnectToLocator(const TArray<FString>& Args, UWorld* World)
```

Comment:
Found this name a little confusing and in a strange location. 
Could we please rename this to "ConsoleCommand_ConnectToLocator" or something to make it distinct from any other connection flow.
As for location, perhaps we need a class for console commands.

Reply:
Done